data/human/sales.txt

```diff
@@ -403,6 +406,7 @@ outfitter "Kraz Basics"
 	"Brig"
 
 outfitter "Kraz Advanced"
+	"Capybara Reverse Thruster"
```

Comment:
Likewise, as the reverse atomic is a Deep Sky special, so should the Capybara be a Delta V special.

```suggestion
```